src/sgame/sg_bot_parse.cpp

```diff
@@ -1403,7 +1403,10 @@ pc_token_list *CreateTokenList( int handle )
 		current->token.intvalue = token.intvalue;
 		current->token.subtype = token.subtype;
 		current->token.type = token.type;
-		current->token.string = BG_strdup( token.string );
+		if ( !BG_strdup( token.string, current->token.string, sizeof( current->token.string ) ) )
```

Comment:
This does not make sense. Size of a pointer is being used as a buffer size

Reply:
You gave me a doubt, so I did a quick try locally:

```
#include <stdio.h>

struct bla { char foo[1024]; };
int main() {
	bla hello;
	printf( "%zu\n", sizeof( hello.foo ) );
}
```

Then: `clang++ test.cpp && ./a.out` prints 1024, as expected.